pfcpiface/grpcsim.go

```diff
@@ -222,19 +222,19 @@ func (u *upf) sim(mode simMode, s *SimModeInfo) {
 			fseID: uint64(n3TEID + i),
 			qfi:   9,
 			ulGbr: 50000,
-			ulMbr: 90000,
+			ulMbr: s.UplinkMBR,
 			dlGbr: 60000,
-			dlMbr: 80000,
+			dlMbr: s.DownlinkMBR,
 		}
 
 		qerN9 := qer{
 			qerID: n9,
 			fseID: uint64(n3TEID + i),
 			qfi:   8,
 			ulGbr: 50000,
-			ulMbr: 60000,
+			ulMbr: s.UplinkMBR,
```

Comment:
Do you not have to update `ulGbr`, as well? What happens if `ulMbr` is less than `ulGbr`?

Reply:
Same as my previous reply (why a `ulGbr` is assigned if the `qfi` is for a non-GBR flow?)

Reply:
FYI, I did tested the traffic with UDP and changed the ulGbr/dlGbr values to different values and confirmed that it has no impact.